### DISCUSSION TITLE
Bookshelf

### DISCUSSION THREAD

**Discussion by lcgithubcode** [sumatrapdfreader/sumatrapdf#3361] [Ideas]
Hi
Thanks for this great piece of SW.

Been using sumatrapdf as my main reader while on windows and would be usefull to have a gallery or bookshelf with our library of books. Something like readera app on android.

Also usefull would be search capability extended to  files in a directory.

**Comment by GitHubRulesOK** [comment]
A gallery would require a whole set of windows (about as much code as adding a GUI editor) So SumtraPDF adds both those functions externally to Windows Explorer.
Currently previewer in a folder allows you to build your own Bookshelf with Search PDFs (there are also Many other 3rd Party variants and for Search slower DnGrep or faster AgentRansack

So search files is via Explorer Gallery function here are 33 files (My BAD I forgot to say **only** look for PDF) that contain Address Book in some way

![image](https://user-images.githubusercontent.com/12754913/226110546-1165eb6e-60c1-45f9-bbc6-3b6803924fe0.png)
